core/src/main/java/io/smallrye/openapi/api/OpenApiConfig.java

```diff
@@ -10,6 +10,7 @@
 
 import io.smallrye.openapi.api.constants.JsonbConstants;
 import io.smallrye.openapi.api.constants.OpenApiConstants;
+import org.jboss.jandex.DotName;
```

Comment:
I am not sure if we want Jandex in the Config ? @MikeEdgar  ? Maybe also use pattern or even just String

Reply:
Yeah, I agree that the config should stick to JDK types. DotName is effectively a string anyway, from a configuration perspective.

Reply:
I think you can remove this import of `DotName`